teuthology/orchestra/remote.py

```diff
@@ -444,6 +444,12 @@ def console(self):
             self._console = getRemoteConsole(self.name)
         return self._console
 
+    @property
+    def uptime(self):
+        proc = self.run(args=['cat', '/proc/uptime'], stdout=StringIO())
+        proc.wait()
+        return float(proc.stdout.read().split(' ')[0])
+
```

Comment:
The whole "connect and then power cycle from the outside" seems wrong.  I've posted PR #1024 that should hopefully supersede this.